resources/views/individuals/partials/about.blade.php

```diff
@@ -11,9 +11,12 @@
             <li>{{ get_language_exonym($language) }}</li>
         @endforeach
     @else
-        @foreach ($individual->languages as $language)
-            <li>{{ get_language_exonym($language) }}</li>
-        @endforeach
+        <li>{{ get_language_exonym($individual->user->locale) }}
+            @if ($individual->user->singed_language)
+                @foreach ($individual->user->singed_language as $language)
+        <li>{{ get_language_exonym($language) }}</li>
+    @endforeach
+    @endif
```

Comment:
There will only be one so there's no need for a loop— also, there's a typo (signed not singed). Thanks @chosww!

Reply:
My bad, I kept making a small mistakes and had to commit small changes multiple times. The have addressed all the issues from your comments.